plugins/hydra_rq_launcher/example/config_redis_ssl.yaml

```diff
@@ -0,0 +1,9 @@
+defaults:
+  - override hydra/launcher: rq
+
+task: 1
+
+hydra:
+  launcher:
+    redis:
+      ssl: true
```

Comment:
I think we don't need an example application for this.
When testing, you can just pass the appropriate equivalent overrides ("hydra.launcher.redis.ssl=true").

Reply:
Changed the test.